app/ydoc-shared/src/ast/mutableModule.ts

```diff
@@ -342,25 +411,40 @@ export class MutableModule implements Module {
 
   /** @internal */
   getToken(token: SyncTokenId): Token
+  /**
+   *
+   */
   getToken(token: SyncTokenId | undefined): Token | undefined
+  /**
+   *
+   */
   getToken(token: SyncTokenId | undefined): Token | undefined {
     if (!token) return token
     if (token instanceof Token) return token
     return Token.withId(token.code_, token.tokenType_, token.id)
   }
 
+  /**
+   *
+   */
   getAny(node: AstId | SyncTokenId): MutableAst | Token {
     return isTokenId(node) ? this.getToken(node) : this.get(node)
   }
 
+  /**
+   *
+   */
   getConcrete(child: RawNodeChild): NodeChild<Ast> | NodeChild<Token> {
     if (isTokenId(child.node))
       return { whitespace: child.whitespace, node: this.getToken(child.node) }
     else return { whitespace: child.whitespace, node: this.get(child.node) }
   }
 
-  /** @internal Copy a node into the module, if it is bound to a different module. */
+  /** @internal */
   copyIfForeign<T extends MutableAst>(ast: Owned<T>): Owned<T>
+  /**
+   *
+   */
   copyIfForeign<T extends MutableAst>(ast: Owned<T> | undefined): Owned<T> | undefined {
```

Comment:
true, i've added comments saying "this is the implementation" elsewhere just to make the plugin happy

Reply:
I think the proper solution would be to just put move the docs before implementation. Lint does not require those for overloads.

Reply:
i totally would have done that if i could! the issue with that is, then the docs seem to disappear from hover tooltips if i do that (because there no longer are docs attached to the overloads)

Reply:
Interestingly, it works in my case (I use vscode):

Definition:

![image](https://github.com/user-attachments/assets/e504a2d7-c6b5-4648-a3f4-d077303bee69)

Usage hover:

![image](https://github.com/user-attachments/assets/bd485183-cccc-42ea-92c2-78e2ca86338f)

Maybe just your caches weren't updated yet?

Reply:
odd, maybe it's only when hovering over the overload declaration itself that it doesn't work